iceoryx_posh/source/roudi/process_manager.cpp

```diff
@@ -267,7 +267,7 @@ bool ProcessManager::addProcess(const RuntimeName_t& name,
     auto offset = rp::BaseRelativePointer::getOffset(m_mgmtSegmentId, m_segmentManager);
     sendBuffer << runtime::IpcMessageTypeToString(runtime::IpcMessageType::REG_ACK)
                << m_roudiMemoryInterface.mgmtMemoryProvider()->size() << offset << transmissionTimestamp
-               << m_mgmtSegmentId;
+               << m_mgmtSegmentId << isMonitored;
```

Comment:
Hehe, this breaks the tests since the RouDiEnvironment does not do monitoring but some tests checks if the keep-alive works as expected. Can you add a parameter like `linkKeepAliveToMonitoring` which is true for regular RouDi and false for the RouDi environment. This line would then become
```cpp
<< m_mgmtSegmentId << (linkKeepAliveToMonitoring ? isMonitored : ALWAYS_KEEP_ALIVE); // constexpr bool ALWAYS_KEEP_ALIVE {true};
```

Reply:
I would dismiss your suggestion when the CI is green.